VolleyPaint/Assets/Scripts/Ability/SlingShot.cs

```diff
@@ -121,7 +122,8 @@ void OnFireButtonUp()
 	{
 		rb.isKinematic = false;
 		Physics.Raycast(Camera.main.ScreenPointToRay(Input.mousePosition), out var hit, 300f, groundMask);
-		Fire(hit.point);
+		//Fire(hit.point);
```

Comment:
I think you need to uncomment this line or else when a player thats not the server tries to use sling shot, it won't appear that they are using it on their screen